### ISSUE TITLE
Rename framerate attribute to sample_frequency

### ISSUE THREAD

**Issue by psomhorst** [EIT-ALIVE/eit_dash#62]
In https://github.com/EIT-ALIVE/eitprocessing/pull/290 we changed the `EITData.framerate` attribute to `EITData.sample_frequency`. This will go in effect probably in [v.1.3.0](https://github.com/EIT-ALIVE/eitprocessing/milestone/17). 

Note that this issue should only be closed after the dash will support v.1.3.0 and later.

**Comment by wbaccinelli** [comment]
@psomhorst Yes, the eitprocessing version is fixed [here](https://github.com/EIT-ALIVE/eit_dash/blob/9ccccf47d0aada25a880efbda85b234d5eee789c/pyproject.toml#L33). When upgrading the version to the new one, this issue has to be addressed.
